routing/DELEGATED_CONTENT_ROUTING_HTTP.md

```diff
@@ -182,61 +158,12 @@ Schema: `bitswap`
 
 The server should respect a passed `transport` query parameter by filtering against the `Addrs` list.
 
-
-#### Write Provider Records
-
-```json
-{
-    "Protocol": "transport-bitswap",
-    "Schema": "bitswap",
-    "Signature": "<signature>",
-    "Payload": "<payload>"
-}
-```
-
-- `Signature`: a multibase-encoded signature of the sha256 hash of the `Payload` field, signed using the private key of the Peer ID specified in the `Payload` JSON
```

Comment:
> - `Signature`: a multibase-encoded signature of the sha256 hash of the `Payload` field, signed using the private key of the Peer ID specified in the `Payload` JSON. Signing details for specific key types should follow [libp2p/peerid specs](https://github.com/libp2p/specs/blob/master/peer-ids/peer-ids.md#key-types), unless stated otherwise.